lexicons/app/bsky/feed/detach.json

```diff
@@ -0,0 +1,27 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.feed.detach",
+  "defs": {
+    "main": {
+      "type": "record",
+      "key": "tid",
+      "description": "Record defining post URIs detached from a root post. The record key (rkey) of the detach record must match the record key of the root post in question, and that record must be in the same repository.",
+      "record": {
+        "type": "object",
+        "required": ["targets", "updatedAt"],
+        "properties": {
```

Comment:
Realized that the `threadgate` record has a reference to the post it's for.

It's deterministic and not really needed right now, but I think we probably want to add it here for:
- symmetry with threadgate records
- in case detach records can ever apply to another type of record